### ISSUE TITLE
Why WP Router?

### ISSUE THREAD

**Issue by distributev** [distributev/gmail-wordpress-mailchimp-integration#5]
Why WP Router is required?

My understanding was that you tried to use the latest V2 rest api but because of a bug in pods.io / REST you had to rollback to V1 rest API. So all the rest stuff should be handled by v1 wp rest.

So why is then WP Router still required? Why both v1 wp rest api and WP Router are used in the same time?

**Comment by mdkass** [comment]
Because I told you that make a custom end point using api v1 is very
difficult.

**Comment by distributev** [comment]
V1 is required for pods.io becuase there is a bug with V2/pods

However both v1 and v2 work on the same time on the same Wordpress installation?

Can be v2 used for the custom endpoint?

Following is a v2 custom endpoint which was developed few days ago for me and works fine.

https://github.com/distributev/edd-software-licences-rest/blob/master/edd-software-licences-rest.php

**Comment by mdkass** [comment]
No, V1 can't be used with v2, and if they are both installed v1 takes priority.
